cfme/cloud/instance.py

```diff
@@ -177,7 +177,12 @@ class OpenStackInstance(Instance):
     STOP = "Stop"
     PAUSE = "Pause"
     RESTART = "Restart"
-
+    # Provider-only shelve options since 5.5
```

Comment:
What does it mean `provider-only` in this context? If it means its only available on the provider, not in CFME, then it shouldn't be here. Please elaborate.